src/pages/tBTC/Bridge/components/DepositDetailsStep.tsx

```diff
@@ -53,108 +33,66 @@ type CommonStepProps = Pick<BridgeProcessStepProps, "onComplete"> & {
-    BridgeProcessStepProps,
-    "txHash" | "onComplete"
-  >
-> = ({ confirmations, requiredConfirmations, txHash, onComplete }) => {
-  const subtitle = `Your Bitcoin deposit transaction requires ${requiredConfirmations} confirmation${
-    requiredConfirmations !== undefined && requiredConfirmations > 1 ? "s" : ""
-  } on the Bitcoin Network before initiating the minting process.`
-
+  { confirmations?: number; requiredConfirmations?: number } & CommonStepProps
+> = ({ confirmations, requiredConfirmations, onComplete }) => {
+  const hasConfirmations = !!confirmations && !!requiredConfirmations
   return (
     <BridgeProcessStep
-      title="Waiting for the Bitcoin Network Confirmations..."
-      chain="bitcoin"
-      txHash={txHash}
-      icon={<BTCConfirmationsIcon />}
-      progressBarColor="brand.500"
-      progressBarValue={confirmations}
-      progressBarMaxValue={requiredConfirmations}
+      loaderLabel="Minting"
+      loaderProgress={
+        0.25 * (hasConfirmations ? confirmations / requiredConfirmations : 1)
+      }
+      headingPrimary="Waiting for the Bitcoin Network Confirmations..."
+      headingSecondary={
+        <BitcoinConfirmationsSummary
+          minConfirmationsNeeded={requiredConfirmations}
+          txConfirmations={confirmations}
+        />
+      }
       isCompleted={Boolean(
         confirmations &&
           requiredConfirmations &&
           confirmations >= requiredConfirmations
       )}
       onComplete={onComplete}
-    >
-      <BitcoinConfirmationsSummary
-        minConfirmationsNeeded={requiredConfirmations}
-        txConfirmations={confirmations}
-      />
-      <BodyMd mt="6" px="3.5" alignSelf="flex-start">
-        {subtitle}
-      </BodyMd>
-    </BridgeProcessStep>
+    />
   )
 }
 
 export const Step2: FC<CommonStepProps> = ({ txHash, onComplete }) => {
   return (
     <BridgeProcessStep
-      title="Minting Initialized"
-      icon={<MintingInitializedIcon />}
-      chain="ethereum"
-      txHash={txHash}
-      progressBarColor="yellow.500"
+      loaderLabel="Minting"
+      loaderProgress={0.25}
+      headingPrimary="Minter Check"
       isCompleted={!!txHash}
       onComplete={onComplete}
-      isIndeterminate
-    >
-      <Box mt="6" px="3.5" alignSelf="flex-start">
-        <BodyMd mb="9">
-          A Minter is assessing the minting initialization. If all is well, the
-          Minter will transfer the initialization to the Guardian.
-        </BodyMd>
-        <BodyMd>
-          Minters are a small group of experts who monitor BTC deposits on the
-          chain.
-        </BodyMd>
-      </Box>
-    </BridgeProcessStep>
+    />
   )
 }
 
 export const Step3: FC<CommonStepProps> = ({ txHash, onComplete }) => {
   return (
     <BridgeProcessStep
-      title="Guardian Check"
-      icon={<GuardianCheckIcon />}
-      chain="ethereum"
-      progressBarColor="green.500"
+      loaderLabel="Minting"
+      loaderProgress={0.5}
```

Comment:
Hmm, loks like loader progress doesn't work properly. For Step 3 it should be at 50%, but it's at 25:

<img width="1014" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/53b34046-d253-4c17-8a56-9af6b5e07473">

I've also had 25 at step1, when it should be 0.

Reply:
I have accidentally left hard coded value, the prop value wasn't passed to the function 😆 Should be fine now
Ref commit: 55f368196007890c1c923524410289ae6c72fae8